openshift/dockerfiles/console-plugin.Dockerfile

```diff
@@ -0,0 +1,32 @@
+ARG BUILDER=registry-proxy.engineering.redhat.com/rh-osbs/rhacm2-yarn-builder:rhel_8_nodejs_18_yarn_builder
```

Comment:
I am not sure this will work (not behind the VPN).

Reply:
ya changed to registry.access.redhat.com/ubi8/nodejs-18:latest